com.chartboost.mediation/Android~/chartboost-mediation-android-bridge/src/main/java/com/chartboost/mediation/unity/UnityBridge.kt

```diff
@@ -57,6 +58,21 @@ class UnityBridge {
             }
         }
 
+        @JvmStatic
+        fun loadBannerAd(listener: ChartboostMediationBannerViewListener): BannerAdWrapper{
+            val size = HeliumBannerAd.HeliumBannerSize.STANDARD;
+            val bannerView = HeliumBannerAd(UnityPlayer.currentActivity,"", size, null);
```

Comment:
So we will always be requesting a Standard banner, regarding of size?

Reply:
Size and placement here are just placeholders. Unity uses the `load(placement, size, listener)` call of `HeliumBannerAd` to send placement and size